modules/ebi-metagenomics/catcontigs/meta.yml

```diff
@@ -0,0 +1,67 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "CATCONTIGS"
+description: MGnify contig taxonomy assignation using CAT"
+keywords:
+  - mags
+  - taxonomy
+  - genomics
+tools:
+  - "catcontigs":
+      description: "CAT/BAT: tool for taxonomic classification of contigs and metagenome-assembled
+        genomes (MAGs)"
+      homepage: "https://github.com/MGXlab/CAT_pack"
+      documentation: "https://github.com/MGXlab/CAT_pack"
+      tool_dev_url: "https://github.com/MGXlab/CAT_pack"
+      doi: "10.1038/s41467-024-47155-1."
+      licence: ["MIT"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - contigs_fasta:
+        type: file
+        description: The fasta file that contains contigs
+        pattern: "*.{fa,fasta}"
+  - - cat_db:
+        type: directory
+        description: Path to directory that contains CAT database files
+  - - taxonomy_db:
+        type: directory
+        description: Path to directory that contains CAT taxonomy files
```

Comment:
There are double dashes `- -` on these

Reply:
To represent tuples, like
```
input:
tuple val(meta), path(thingy)
path(db)
```
used to be represented as a flat schema, now it should be represented as

```
input:
 - #inputone
  - meta
  - thingy
 - db
```

Reply:
oooh fancy!! I didn't know, thank you